environments/kubernetes/manifests/observatorium-api-thanos-querier-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: observatorium-api-thanos-querier
+  name: observatorium-api-thanos-querier
+  namespace: observatorium
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: observatorium-api-thanos-querier
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: observatorium-api-thanos-querier
+        app.kubernetes.io/tracing: jaeger-agent
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                  - observatorium-api-thanos-querier
+              namespaces:
+              - observatorium
+              topologyKey: kubernetes.io/hostname
+            weight: 100
+      containers:
+      - args:
+        - query
+        - --query.replica-label=replica
+        - --query.replica-label=ruler_replica
+        - --query.replica-label=prometheus_replica
+        - --web.external-prefix=https://observatorium.api/ui/v1/metrics
```

Comment:
This is the actual parameter I wanted to add. To prevent any broken changes for existing environments, I created a new deployment,

Reply:
Should we use our actual DNS name here?

Reply:
👍